transmit-lib/src/cmr/transmit/config.clj

```diff
@@ -36,6 +36,10 @@
   "The test user's password"
   "Password101")
 
+(def mock-urs-token
```

Comment:
I wouldn't put test code into the library - why is this needed? It looks like it's being checked in `assert-bearer-token`, but I don't entirely follow.

I think what's supposed to happen is transmit calls URS to get a bearer token, then in subsequent calls it passes that bearer token in.

So mock-urs should be the one that has some mock-token rather than the transmit lib.